spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -253,7 +253,9 @@ object DatabaseTablesDAO {
       val value = body(body.keys.toList(j - i - 1))
       preparedStatement.setObject(j, value)
     }
-    val _ = preparedStatement.executeUpdate()
+    val result = preparedStatement.executeQuery()
+    result.next()
+    result.getString(1)
```

Comment:
What's this returning? is it guaranteed to be present?

Reply:
It's the value of the id of the table, it always returns a String (::text) value of the id. We have to return it to make known React-Admin that the row was created and React-Admin automatically redirects from the /table/create page to the /table/:id page